koji/collection-mash-split.py

```diff
@@ -160,6 +165,16 @@ def createrepo(self, path, comps=None, checksum=None):
         self.logger.debug("running %s" % cmd)
         kobo.shortcuts.run(cmd)
 
+    def inject_modulemd_yml(self, path, modulemd_yaml):
+        """Generate modular metadata and inject it.
+
+        @param path: path to repository
+        @type path: str
+        @param modulemd_yaml: metadata template
+        @type modulemd_yaml: str
+        """
+        pass
```

Comment:
obviously a stub. I don't think running `ansible` in here is a good idea, so I'd probably re-implement the playbooks in pure python.